latest/src/app/documentation/demos/search-field/search-field.demo.html

```diff
@@ -10,11 +10,13 @@ <h3 id="examples">Code &amp; Examples</h3>
                 compatible with the existing <code class="clr-code">ClrInputContainer</code> and forms.
             <h3>Angular Directive</h3>
             <form clrForm>
-                <clr-input-container>
-                    <label>Search for any value</label>
-                    <input clrInput clrSearch type="text" name="search" [(ngModel)]="value"/>
-                  </clr-input-container>
-            </form>
+                <div class="clr-col-md-4">
```

Comment:
please look at numericfield.demo.html
there we have also different breakpoints for the input fields and set them directly to the <input>